scripts/opencontrail-kubelet/opencontrail_kubelet/plugin.py

```diff
@@ -231,4 +240,8 @@ def main():
         sys.exit(1)
 
 if __name__ == '__main__':
-    main()
+    try:
```

Comment:
OK, I took it out. I left it because, I thought it would catch exception if
we launch the plugin directly from the command line..

Thx

On Wed, Jul 8, 2015 at 4:32 PM, Pedro Marques <notifications@github.com>
wrote:

> In scripts/opencontrail-kubelet/opencontrail_kubelet/plugin.py
> <https://github.com/Juniper/contrail-kubernetes/pull/13#discussion_r34210539>
> :
>
> > @@ -231,4 +240,8 @@ def main():
> >          sys.exit(1)
> >
> >  if __name__ == '__main__':
> > -    main()
> > +    try:
>
> This diff doesn't do anything.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Juniper/contrail-kubernetes/pull/13/files#r34210539>.
>